### PR TITLE
Add missing assertion for test_route_with_colon_first

### PR DESCRIPTION
**Pull request by aditya-kapoor**

### CODE REVIEW

File: actionpack/test/controller/routing_test.rb [hunk 1]
```diff
@@ -334,10 +334,9 @@ def test_ignores_leading_slash
   end
 
   def test_route_with_colon_first
-    rs.draw do
-      get '/:controller/:action/:id', :action => 'index', :id => nil
-      get ':url', :controller => 'tiny_url', :action => 'translate'
-    end
```
Comment (senny) [inline actionpack/test/controller/routing_test.rb]: why do we have to change the setup like this? I'd rather keep the existing setup than potentially changing the tests meaning.

Comment (aditya-kapoor) [inline actionpack/test/controller/routing_test.rb]: :ok: reverting the definition.

**Comment by aditya-kapoor** [comment]
@senny reverted the definition... :smiley:

File: actionpack/test/controller/routing_test.rb [hunk 2]
```diff
@@ -335,9 +335,11 @@ def test_ignores_leading_slash
 
   def test_route_with_colon_first
     rs.draw do
-      get '/:controller/:action/:id', :action => 'index', :id => nil
-      get ':url', :controller => 'tiny_url', :action => 'translate'
+      get '/:controller/:action/:id', action: 'index', id: nil
+      get ':url', controller: 'content', action: 'translate'
```
Comment (senny) [inline actionpack/test/controller/routing_test.rb]: why change the name of the controller?

Comment (aditya-kapoor) [inline actionpack/test/controller/routing_test.rb]: It is raising an error of initialized constant `TinyUrlController`.

**Pull request merged**